build/pmm-client-docker/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN install -d -o pmm-agent -g pmm-agent -m 0775 /usr/local/percona/pmm2
 
 RUN tar -zxpf /tmp/pmm2-client.tar.gz -C /tmp \
     && cd /tmp/pmm2-client-* \
-    && env PMM_USER=pmm-agent PMM_GROUP=pmm-agent ./install_tarball \
+    && env PMM_USER=pmm-agent PMM_GROUP=root ./install_tarball \
```

Comment:
Cherry-picked from https://github.com/Percona-Lab/pmm-submodules/pull/1233 (already merged into PMM-2.0), can be reset during the next PMM-2.0 reset